Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -22,27 +24,34 @@ public MixedRealityEditorSettings()
         private const string IgnoreKey = "_MixedRealityToolkit_Editor_IgnoreSettingsPrompts";
         private const string SessionKey = "_MixedRealityToolkit_Editor_ShownSettingsPrompts";
 
-        private static string mixedRealityToolkit_RelativeFolderPath = string.Empty;
-
+        [Obsolete("Use the 'MixedRealityToolkitFiles' APIs.")]
         public static string MixedRealityToolkit_AbsoluteFolderPath
         {
             get
             {
-                if (string.IsNullOrEmpty(mixedRealityToolkit_RelativeFolderPath))
+                if (MixedRealityToolkitFiles.AreFoldersAvailable)
                 {
-                    if (!FindDirectory(Application.dataPath, "MixedRealityToolkit", out mixedRealityToolkit_RelativeFolderPath))
+#if UNITY_EDITOR
+                    if (MixedRealityToolkitFiles.MRTKDirectories.Count() > 1)
                     {
-                        Debug.LogError("Unable to find the Mixed Reality Toolkit's directory!");
+                        Debug.LogError($"A deprecated API '{nameof(MixedRealityEditorSettings)}.{nameof(MixedRealityToolkit_AbsoluteFolderPath)}' " +
```

Comment:
Is this to inform of deprecated method use in the core toolkit or is this something that app / extension code may hit?

Reply:
app/extension code may potentially hit this, although I don't know how many would or do rely on MixedRealityToolkt_AbsoluteFoldersPath currently.

---
In reply to: [276630039](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3991#discussion_r276630039) [](ancestors = 276630039)